.github/workflows/test.yml

```diff
@@ -0,0 +1,453 @@
+# This file was auto-generated by .evergreen/make-github-actions
+# at 2021-08-21T15:14:55Z
+---
+name: Run Mongoid Tests
+'on':
+- push
+- pull_request
+jobs:
+  build:
+    name: "${{matrix.ruby}} driver-${{matrix.driver}} mongodb-${{matrix.mongodb}}
+      ${{matrix.topology}}"
+    env:
+      CI: true
+      TESTOPTS: "-v"
+    runs-on: ubuntu-latest
+    continue-on-error: "${{matrix.experimental}}"
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - mongodb: '4.4'
+          ruby: ruby-3.0
+          topology:
+          task: test
+          driver: current
+          rails:
+          i18n:
+          gemfile: Gemfile
+          experimental: false
```

Comment:
I suppose this defines all builds in the matrix individually (instead of using `matrix.ruby` and `matrix.topology` arrays to have GH actions generate the matrix automatically) since it's auto-generated. No need to change this since we don't expect people to make manual changes in this file.

I'll note that it makes it a little more difficult to figure out what's being tested by looking at the file, but that's a problem that can be solved another day.

Reply:
Yep you got it.